doc/rbd/rbd-mirroring.rst

```diff
@@ -305,10 +305,9 @@ For example::
 .. tip:: ``rbd-mirror`` tunables are set by default to values suitable for
    mirroring an entire pool.  When using ``rbd-mirror`` to migrate single
    volumes been clusters you may achieve substantial performance gains
```

Comment:
not related to this change, but I can't understand "been" word here. May be it should have been "between"?

Reply:
Fixed.